packages/devtools-environment/index.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+const flag = require("./test-flag");
+
+function isBrowser() {
+  return typeof window == "object";
+}
+
+export function isDevelopment() {
+  if (isBrowser()) {
+    const href = window.location ? window.location.href : "";
+    return href.match(/^file:/) || href.match(/localhost:/);
```

Comment:
I know you just copied over but couldn't this be achieved with 1 match call since regex supports or?

Reply:
true... but i think this is easier to read